### ISSUE TITLE
 is this project still active? 

### ISSUE THREAD

**Issue by Magalame** [Daniel-Diaz/matrix#61]

**Comment by octopuscabbage** [comment]
Not really. I used to make contributions but I haven't picked up haskell in years. I don't know if @Daniel-Diaz works on this either.

**Comment by Daniel-Diaz** [comment]
Since some time ago I haven't had time to actively work in this package. I have looked for other maintainers, but didn't succeeded. Right now I am only doing minimal maintainance.
